make-all.bash

```diff
@@ -1,12 +1,6 @@
 #!/bin/bash -e
 
 # Build the syncthing library
-
-./make-go.bash arm
```

Comment:
Yes, there is no guarantee that the source corresponds to the given binary. This is a guarantee F-Droid wishes to provide. The apk & source can be downloaded for every application. The only binary upstream they trust is Debian (as far as I know).